### ISSUE TITLE
kdk up fails because of incompatable Dokan library

### ISSUE THREAD

**Issue by erikburgess** [cisco-sso/kdk#108]
kdk up fails on Windows because keybase requires Dokan 1.2.0 and the kdk expects 1.1.0. Downgrading Dokan to 1.1.0 causes keybase filesystem integration to be disabled.

```
The system cannot find the file C:\Program Files\Dokan\Dokan Library-1.1.0\sample\mirror\mirror.exe.
"Unrecognized parameter start. You must pass either start or stop"
INFO[0019] An exited KDK container exists
```

Keybase Client: 2.11.0-20181203203419+ec71fb1ab3

**Comment by dcwangmit01** [comment]
Another data point.
After upgrade of docker for windows 2.0, the dokan mirror.exe disappears.  We need to install this and then reboot https://github.com/dokan-dev/dokany/releases/download/v1.1.0.2000/DokanSetup_redist.exe

**Comment by venkats6691** [comment]
Unable to run keybase with dokan 1.1.0, It shows that it requires only dokan 1.2.0

**Comment by dcwangmit01** [comment]
Please review:https://github.com/cisco-sso/kdk/pull/114

**Comment by dcwangmit01** [comment]
Should be fixed in release 1.4.1

`kdk update`
